examples/readme.md

```diff
@@ -0,0 +1,25 @@
+# `dattri` examples
+This folder contains bite-sized examples which can help users to build their own application by `dattri`.
+
+## Noisy label detection
+This section contains using different attributors to detect the noisy label in various datasets.
```

Comment:
This section contains examples using different attributors to detect noisy labels in various datasets.